src/blockdata/transaction.rs

```diff
@@ -737,7 +737,7 @@ impl error::Error for NonStandardSighashType {}
 
 /// Legacy Hashtype of an input's signature
 #[deprecated(since = "0.28.0", note = "Please use [`EcdsaSighashType`] instead")]
-pub type SigHashType = EcdsaSighashType;
+pub type SighashType = EcdsaSighashType;
```

Comment:
Same note here

Reply:
Will do, thanks.

Reply:
Removed patch: `eaf2040 Rename SigHashType -> SighashType`, win for separate patches!